apps/api/src/app/events/dtos/trigger-event-request.dto.ts

```diff
@@ -20,7 +20,7 @@ export class TopicPayloadDto {
   @ApiProperty()
   id: TopicId;
   @ApiProperty()
-  type: TriggerRecipientsTypeEnum;
+  type: TriggerRecipientsTypeEnum.TOPIC;
```

Comment:
shouldn't you assign the value then here if it will be always a topic type?

Reply:
I am not 100% sure but doesn't this allow only that entry from the enum?